exercises/practice/bob/tests/Tests.elm

```diff
@@ -127,8 +127,8 @@ tests =
             test "on multiple line questions" <|
                 \() ->
                     Expect.equal
-                        "Whatever."
-                        (Bob.hey "\nDoes this cryogenic chamber make me look fat?\nno")
+                        "Sure."
```

Comment:
This looks like the instructions would have to change too?

Reply:
Nope, now it ends with a question mark, so the expected answer is different, as per the instructions.
I'm not sure what motivated this particular change, you'd have to check on the problem-specs repo.